### ISSUE TITLE
Using the same "Big" image prevents the preview from paging correctly

### ISSUE THREAD

**Issue by DBosley** [lukasz-galka/ngx-gallery#33]
using the example `this.galleryImages` but editing it as shown below prevents the large image preview from loading the other images when pressing the left or right arrow:

```
this.galleryImages = [
  {
    small: 'https://placehold.it/900x696',
    medium: 'https://placehold.it/900x696',
    big: 'https://placehold.it/900x696'
  },
  {
    small: 'https://placehold.it/900x696',
    medium: 'https://placehold.it/900x696',
    big: 'https://placehold.it/900x696'
  },
  {
    small: 'https://placehold.it/900x696',
    medium: 'https://placehold.it/900x696',
    big: 'https://placehold.it/900x696'
  }
];
```

**Comment by lukasz-galka** [comment]
Hi @DBosley which version are you using?

**Comment by DBosley** [comment]
I installed the library for the first time yesterday with all the latest released angular dependencies.

**Comment by lukasz-galka** [comment]
fixed in version 1.1.5, you can see this in https://lukasz-galka.github.io/ngx-gallery-demo/#using-the-same-images
